plugin/python_dependency_usage/config.go

```diff
@@ -0,0 +1,11 @@
+package python_dependency_usage
```

Comment:
I think we need a package name that conforms to Go conventions
https://go.dev/blog/package-names

Reply:
how about -
- `pydeps` -> python dependency related analysis will go here, this implies we'll have plugins like godeps, javadeps etc
-`deps....?` -> all about dependencies will go here, will cover all languages in single plugin
- `depusage` -> dependency usage will go here, will cover all languages in single plugin

Reply:
@OmkarPh `depsusage` seems fine.